lablib/renderers/__init__.py

```diff
@@ -2,10 +2,12 @@
 
 from .basic import BasicRenderer, Burnin, Codec
 from .slate_render import SlateRenderer
+from ._base import RendererBase
```

Comment:
for consistency i would name the base module `base.py` instead of `_base.py`.
i understand it's for indicating only private access.

Reply:
This is addressed, thanks for the suggestion.